hi-authen-fe/Gruntfile.js

```diff
@@ -0,0 +1,42 @@
+/* 
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+module.exports = function (grunt) {
+    var dataDir = 'D:/Hien/hiperform/wrapper/httpserver-win64/app/authen/';
```

Comment:
Không nên để cấu hình local ở đây, project to quá chắc phải chuẩn hóa lại nhiều thì mới bắt đầu review pull request hiệu quả được :D

Reply:
kaka, chắc hẳn là vậy rồi bác @dominhhien ạ, có nên xé lẻ từng con nhỏ ra hay không bác nhỉ? mà file này thì đúng là từ thời bác rồi

Reply:
Theo anh là đẩy những phần code lên thôi, phần thư viện JSS, CSS nên để ignore

Reply:
Dạ tại em định làm CI/CD luôn nên nếu không đẩy lên thì hơi khó cho việc đó anh @dominhhien  ạ.